apps/trading/components/footer/footer.tsx

```diff
@@ -11,7 +11,10 @@ export const Footer = () => {
   const setNodeSwitcher = useGlobalStore(
     (store) => (open: boolean) => store.update({ nodeSwitcherDialog: open })
   );
-  const { blockDiff, datanodeBlockHeight } = useNodeHealth();
+  const { blockDiff, datanodeBlockHeight, datanodeVegaTime } = useNodeHealth();
+  const blockUpdateMsLatency = datanodeVegaTime
+    ? Date.now() - datanodeVegaTime.valueOf()
```

Comment:
suggestion (non blocking): using getTime is a little more 'standard' javascript

Reply:
done